sdk/include/psdk/uxtheme.h

```diff
@@ -15,8 +15,7 @@ extern "C" {
 #define DTBG_COMPUTINGREGION        0x00000010
 #define DTBG_MIRRORDC               0x00000020
 #define DTT_GRAYED                  0x00000001
-
-/* DTTOPTS.dwFlags bits */
+//Vista+
```

Comment:
```suggestion
// Vista+
```
And same comment as above.